webservices/legal_docs/load_legal_docs.py

```diff
@@ -377,55 +377,80 @@ def get_mur_names(mur_names={}):
                 mur_names[row[1]] = row[2]
     return mur_names
 
-def process_mur(mur):
+
+def get_documents(td_text, bucket):
+
+    documents = []
+    for index, file_name in enumerate(re.findall(r"/disclosure_data/mur/([0-9_A-Z]+)\.pdf", td_text)):
+        logger.info("Loading file %s.pdf", file_name)
+        pdf_key = 'legal/murs/%s.pdf' % file_name
+        pdf_text, pdf_size, pdf_pages = process_mur_pdf(file_name, pdf_key, bucket)
+        pdf_url = '/files/' + pdf_key
+        document = {
+            "document_id": index + 1,
+            "size": pdf_size,
+            "pages": pdf_pages,
```

Comment:
We can get rid of this field as we don't use it (and don't have it for current MURs).

Reply:
Sounds good. I also realized the size should be labeled `length` for consistency.